packages/blade/src/components/Alert/__tests__/Alert.ssr.test.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { Alert } from '../';
+import renderWithSSR from '~src/_helpers/testing/renderWithSSR.web';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
```

Comment:
We should not mock the console tho 🤔 let us atleast see if there are errors on console. 

Saying this because there are certain warning and errors only happens when you do SSR. 
For example using useLayoutEffect on SSR react will throw an warning, thus we need `useIsomorphicLayoutEffect` etc.

Reply:
we need to do a cleanup for everything for that. Also, we can remove these from the top level and move to the test where we test the negative test cases otherwise the whole console gets filled with a negative test where we check `toThrow` things. Created #930 to track this